probe-rs/src/memory/mod.rs

```diff
@@ -2,8 +2,24 @@ use crate::error::Error;
 
 use scroll::Pread;
 
+/// Call to {0} with data.len() not a multiple of {1}
```

Comment:
```suggestion
/// {0} was called with data length that is not a multiple of {1}.
```

I think I would prefer named fields here, just to make the documentation part a bit more readable, but I guess it's fine unless proven otherwise :)

Reply:
This should now be fixed.